db.php

```diff
@@ -1,11 +1,29 @@
 <?php
-$db = new PDO ('mysql:host=db; dbname=sami-collection','root', 'password');
-$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
-$query =  $db->prepare("SELECT `name`, `designer`, `style`, `year_released` FROM `items` ORDER BY `id`;");
-$query->execute();
-$results = $query->fetchAll();
+function getDb(): object {
+    return new PDO ('mysql:host=db; dbname=sami-collection','root', 'password');
+}
+function getItems(object $db): array {
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $query =  $db->prepare("SELECT `name`, `designer`, `style`, `year_released`, `image` FROM `items` ORDER BY `id`;");
+    $query->execute();
+    return $query->fetchAll();
+}
+function arraySplit(array $items) : string{
+    $result = '';
+    foreach ($items as $item){
+        $result .= '<div><img src="'. $item['image'] . '" alt="garment image"></div>';
+        $result .= '<div>' . $item['name'] . '</div>';
+        $result .= '<div>' . $item['designer'] . '</div>';
+        $result .= '<div>' . $item['style'] . '</div>';
+        if($item['year_released'] <= date("Y")) {
+            $result .= '<div>' . $item['year_released'] . '</div>';
+        } else { $result = 'select a year that isn\'t in the future';}
+    }
+    return $result;
+}
 
-echo '<pre>';
-var_dump($results);
-echo '</pre>';
+
+//$db = getDb();
```

Comment:
Should these be commented out?
Returns an error as they are.

Reply:
Now I understand what you meant by this, please uncomment the items to check the output as I had to comment them out so that they didn't interfere with doing the Unit Test

Reply:
Yeah, works when they're uncommented 👍

Reply:
Could you set the variables in the index.php file instead? 
That way you might be able to run tests without effecting functionality by commenting out

Reply:
This is certainly something I will look into; however, I won't get hung up on it as it is unlikely i'd be unit testing this while it is live.

I appreciate it is a level of inconvenience but if you're able to comment out that bit to do the tests it would be much appreciated.